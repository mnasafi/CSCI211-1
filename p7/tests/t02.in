echo t02: Insert "Hello world." into tree. Print nothing.
insert Hello world.
