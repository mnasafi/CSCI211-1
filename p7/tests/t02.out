t02: Insert "Hello world." into tree. Print nothing.
